nimbus/examples/experiment.rs

```diff
@@ -216,10 +217,11 @@ fn main() -> Result<()> {
         ("show-experiments", _) => {
             println!("======================================");
             println!("Printing all experiments (regardless of enrollment)");
-            nimbus_client
-                .get_all_experiments()?
-                .iter()
-                .for_each(|e| println!("Experiment: {}", e.slug));
+            nimbus_client.get_all_experiments()?.iter().for_each(|e| {
+                match e {
+                    Experiment::V1(e) => println!("Experiment: {}", e.slug),
+                };
```

Comment:
A lot of this may be just my lack of experience with Rust and knowing what good approaches might be here. Your suggestions sound a lot simpler than having to match enum variants everywhere and keeps the code that matters in fewer places.  Let me take a shot at refactoring this to be more in line with what you are suggesting.